AdobeStockImageAdminUi/view/adminhtml/web/js/media-gallery.js

```diff
@@ -17,43 +17,54 @@ define([
          * @param {String} path
          */
         locate: function (path) {
+            var i,
```

Comment:
Are all these variables necessary? Is there a way to reduce number of variables used in this function?

Reply:
Due to the 'hoist all `var`s' to the top eslint rule, unfortunately not. If it were up to me I would eliminate this rule and instead declare vars in each relevant scope; then that would allow for reducing the number of vars in this particular case (though they would simply move to the relevant block scopes where they are used).